### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/get-started/viam.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Viam in 3 minutes"
-linkTitle: "Viam in 3 minutes"
+title: "A tour of Viam"
+linkTitle: "Tour of Viam"
```
Comment (JessamyT) [inline docs/get-started/viam.md:3]: What if we call this "What is Viam?" I think we did that back in the day...what was the reason for not doing this? I think it's a frequently asked question and is what many users might be wondering when they land on our docs site (as well as other site). I often find myself wondering "But what actually *is* xyz tech company/product" on different docs sites.

File: docs/get-started/viam.md [hunk 2]
```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
 images: ["/general/understand.png"]
 ---
 
-Viam is a complete software platform that supports every step of your {{< glossary_tooltip term_id="smart-machine" text="smart machine">}} development lifecycle.
+Viam is a software platform that makes it easy to work with hardware and software.
```
Comment (JessamyT) [inline docs/get-started/viam.md:18]: I like this sentence. It's easy to read and gives straightforward overarching context.

File: docs/get-started/viam.md [hunk 3]
```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
...
+At the core of Viam is the open-source `viam-server` executable which:
 
-## Plan your machine
+- Creates, configures, and maintains any machine on which it is installed
```
Comment (JessamyT) [inline docs/get-started/viam.md:21]: I fear that this section gets too technical and detailed too quickly.

However, who is our target audience for this particular bit?

File: docs/get-started/viam.md [hunk 4]
```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
...
+- Securely handles all communications.
+- Runs drivers, custom code, and any other software.
+- Accepts API requests.
+- Runs {{< glossary_tooltip term_id="service" text="services">}} like computer vision, motion planning, data management, machine learning, and more.
 
-When using Viam, this is what you'll need to know to plan your machine:
+`viam-server` **runs on Linux and macOS** and supports a wide variety of popular systems, including:
 
-<img src="https://assets-global.website-files.com/62fba5686b6d47fe2a1ed2a6/633d91b848050946efcf0690_viam-overview-illustrations-build.svg" alt="A diagram of machine parts and software" class="alignright" style="width:250px;"></img>
+{{< board-carousel >}}
```
Comment (JessamyT) [inline docs/get-started/viam.md:29]: Hmm scrolling isn't working in staging

File: docs/get-started/viam.md [hunk 5]
```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
 images: ["/general/understand.png"]
 ---
 
-Viam is a complete software platform that supports every step of your {{< glossary_tooltip term_id="smart-machine" text="smart machine">}} development lifecycle.
+Viam is a software platform that makes it easy to work with hardware and software.
+At the core of Viam is the open-source `viam-server` executable which:
```
Comment (JessamyT) [inline docs/get-started/viam.md:19]:
```suggestion

At the core of Viam is the open-source `viam-server` executable which:
```
See comment below but, at the very least I think it'd help to split this into a second paragraph so they digest the first simple sentence first

File: docs/get-started/viam.md [hunk 6]
```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
...
+- Runs drivers, custom code, and any other software.
+- Accepts API requests.
+- Runs {{< glossary_tooltip term_id="service" text="services">}} like computer vision, motion planning, data management, machine learning, and more.
 
-When using Viam, this is what you'll need to know to plan your machine:
+`viam-server` **runs on Linux and macOS** and supports a wide variety of popular systems, including:
 
-<img src="https://assets-global.website-files.com/62fba5686b6d47fe2a1ed2a6/633d91b848050946efcf0690_viam-overview-illustrations-build.svg" alt="A diagram of machine parts and software" class="alignright" style="width:250px;"></img>
+{{< board-carousel >}}
+<br>
```
Comment (JessamyT) [inline docs/get-started/viam.md:31]: Ahh cool I think the concrete examples will help it click for people. I think we can fit just a smidge more broad-strokes explanation of what Viam is first, and then also include some sort of transition to the examples like "Here are some examples to show how you can use Viam in your projects..."

I like that people won't have to scroll too far before they see concrete things, but I think we just need a little more first.

File: docs/get-started/viam.md [hunk 7]
```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
...
+**Collect and upload data to the cloud**
+
+You can configure the camera to capture a photo of the food bowl periodically and automatically sync these photos to the cloud.
+
+{{% /tablestep %}}
+{{% tablestep link="/services/ml/"%}}
+{{<imgproc src="/get-started/bowl-label.jpg" class="fill alignright" resize="600x" style="max-width: 200px" declaredimensions=true alt="Screenshot of food bowl pictures with labels">}}
+**Use collected data to build data sets and train and Machine Learning models**
+
+Having captured pictures of the food bowl, you can organize them into a dataset and label each as empty or full.
```
Comment (JessamyT) [inline docs/get-started/viam.md:66]: I want to take a stab at a different layout (not sure what yet) and try to reduce the word count a bit through these examples. I fear we might lose people partway through.

File: docs/get-started/viam.md [hunk 8]
```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
...
+
+{{% /tablestep %}}
+{{% tablestep link="/build/program/"%}}
+
+**Write code to control the machine using Python, Go, C++, or other programming languages**
+
+You can now program your pet feeder to feed your pet at different times of the day and you can use the vision service to programmatically know if the food bowl is full or empty.
+If the food bowl is empty, use the Motor API to turn on the motor for 30 seconds, which will fill up the bowl:
+
+{{< expand "See the entire robot logic in 42 lines" >}}
```
Comment (JessamyT) [inline docs/get-started/viam.md:85]: Nice I like this clickable element and the 42 line callout

File: docs/get-started/viam.md (hunk 7 above)
Comment (npentrel) [reply docs/get-started/viam.md:67]: Go for it

**Pull request closed without merging**